src/components/layout/Layout.js

```diff
@@ -60,7 +61,26 @@ const Content = styled(KoroSection).attrs({
   color: 'fog'
 })``;
 
-const Layout = ({ children, hero }: Props) => (
+const HeroContent = styled(KoroSection).attrs({
+  bottom: true,
+  color: 'white'
+})`
+  > div {
+    margin-top: 3em;
+    margin-bottom: 3em;
```

Comment:
Needs little bit more margin in bottom in order to make intro koro symmetric vertically.

<img width="986" alt="screenshot 2018-12-13 at 7 55 55" src="https://user-images.githubusercontent.com/1412709/49918635-8e4c5f80-feac-11e8-9aff-4647633c690d.png">